db/migrate/20180731124028_add_password_checker_to_rusers.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordCheckerToRusers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :rusers, :password_checker, :integer, default: 0
```

Comment:
These could actually be in the same migration!

Reply:
Sorry, I could not understand what I need to change.

Reply:
There are two migrations. First is to delete password_checker from Users.
Second is to add password_checker to RUsers.

Reply:
Oh just that you can perform both steps in the same migration file. They needn't be separated... Although it's not a big deal, over the lifetime of the project I think we should try not to have too many migration files. Mind changing this? Thanks!

Reply:
I will change it

Reply:
Thanks for explanation

Reply:
Can you please add link to the code for example? I tried a lot searching on net and on plots2 migration files. We can have different dbms queries on same table in single migration. But I could not find a migration with two different tables involved. In this pr, we are involved with two tables Rusers(removal of password checker) & Users(addition of password checker).

Reply:
I believe it'll work to do:

```ruby
def change
  remove_column :users, :password_checker, :integer
  add_column :rusers, :password_checker, :integer, default: 0
end
```

I don't have an example but have you tried it? I think the table name is just specified in the one line, no? But honestly it's not a big deal, just a slight preference, so if you find that it doesn't work, 2 files is fine!

Reply:
Thanks a lot. I was thinking other type of solution which was giving errors.
Thanks it worked.